deepchecks/nlp/context.py

```diff
@@ -375,7 +375,7 @@ def get_scorers(self,
             else:
                 scorers = scorers or get_default_scorers(TabularTaskType.BINARY, use_avg_defaults)
         elif self.task_type == TaskType.TOKEN_CLASSIFICATION:
-            scorers = []  # TODO: Complete that
+            scorers = get_default_token_scorers(scorers, span_aligner, use_avg_defaults)
```

Comment:
Create two different paths here, only use get_default_token_scorers if no scorers are provided.
If scorers are provided process them here or create a different function for that
Currently get_default_token_scorers is hard function to follow